Java/AutoML/automl-component/src/main/java/com/automl/component/model/Namespace.java

```diff
@@ -0,0 +1,88 @@
+package com.automl.component.model;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+/**
+ * @author wangqingwei
+ * Created on 2022-08-13
+ */
+public class Namespace {
+    private long id;
+    private String name;
+    private String description;
+    private long ownerId;
+    private long createId;
```

Comment:
The same as above, do you mean creatorId here?

Reply:
The person who created the namespace ,   if you think ownerId and createId is same! 
i can remove ownerId,  keep createId.   
or you think this 'createId' name error?

Reply:
Yes, use creatorId instead of createId. As for ownerId, if they are the same, you can choose to delete one of them.